wrappers/parking_lot/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "shuttle-parking_lot"
+version = "0.0.1"
+edition = "2024"
+
+[features]
+shuttle = [ "dep:shuttle-parking_lot-impl" ]
+
+arc_lock = [ "parking_lot/arc_lock", "shuttle-parking_lot-impl?/arc_lock" ]
```

Comment:
Could we please put the features [in the same order](https://github.com/Amanieu/parking_lot/blob/master/Cargo.toml) as they appear in the `parking_lot` crate:

arc_lock
owning_ref
nightly
deadlock_detection
serde
send_guard
hardware-lock-elision

Reply:
Yes, I think the list here is alphabetical